launch-game.cmd

```diff
@@ -8,11 +8,22 @@ if %INCLUDE_DEFAULT_MODS% neq "True" goto launch
 set MOD_SEARCH_PATHS=%MOD_SEARCH_PATHS%,./mods
 
 :launch
-cd engine
+set TEMPLATE_DIR=%CD%
+if not exist %ENGINE_DIRECTORY% goto noengine
+cd %ENGINE_DIRECTORY%
+if not exist OpenRA.Game.exe goto noengine
+
 OpenRA.Game.exe Game.Mod=%MOD_ID% Engine.LaunchPath="%TEMPLATE_LAUNCHER%" "Engine.ModSearchPaths=%MOD_SEARCH_PATHS%"  "%*"
 if %errorlevel% neq 0 goto crashdialog
-cd ..
+cd %TEMPLATE_DIR%
+exit /b
+
+:noengine
+echo Required engine files not found.
+echo Run `make all` in the mod directory to fetch and build the required files, then try again.
+pause
 exit /b
```

Comment:
Fixed this and the other dedicated/utility scripts by instead not changing into the engine directory in this case.